src/StudentOrganizer.Core/Models/User.cs

```diff
@@ -55,6 +55,14 @@ public User()
 		{
 		}
 
+		public void UpdateData(string email, string password, string salt, string firstName, string lastName)
```

Comment:
UpdateData i konstruktor mają w sobie taki sam kod także można to wrzucić do jednej funkcji